app/models/bulkrax/oai_entry.rb

```diff
@@ -64,7 +64,8 @@ def find_collection_ids
         self.collection_ids << collection.id if collection.present? && !self.collection_ids.include?(collection.id)
       else # All - collections should exist for all sets
         sets.each do |set|
-          c = Collection.find_by(work_identifier => importerexporter.unique_collection_identifier(set.content))
+          # c = Collection.find_by(work_identifier => importerexporter.unique_collection_identifier(set.content))
```

Comment:
Can we remove this if we don't need it?

Reply:
ya I have removed it now. don't know why I kept it in in the first place.